### PR TITLE
changedata request and event row

### PR DESCRIPTION
**Pull request by xiongjiwei**

### CODE REVIEW

File: proto/cdcpb.proto [hunk 1]
```diff
@@ -87,6 +87,7 @@ message Event {
         // and represents the expiration time of this row.
         // Absolute time, seconds since Unix epoch.
         uint64 expire_ts_unix_secs = 8;
+        uint64 txn_source = 9;
```
[the remark this answers is not in the document]
Comment (xiongjiwei) [reply proto/cdcpb.proto:91]: done

File: proto/cdcpb.proto [hunk 2]
```diff
@@ -164,6 +165,7 @@ message ChangeDataRequest {
     }
 
     KvAPI kv_api = 11;
+    bool filter_loop = 12;
```
[the remark this answers is not in the document]
Comment (xiongjiwei) [reply proto/cdcpb.proto:170]: done

File: proto/cdcpb.proto (hunk 1 above)
[the remark this answers is not in the document]
Comment (overvenus) [reply proto/cdcpb.proto:91]: Can we remove it? Since we only support bi-replication.

[the remark this answers is not in the document]
Comment (xiongjiwei) [reply proto/cdcpb.proto:91]: no, for bi-replication we still need a variable to store 0/non-0

**Review by overvenus — COMMENTED** [review]
LGTM

**Review by asddongmen — APPROVED** [review]

**Pull request merged**